example.yml

```diff
@@ -3,8 +3,8 @@
 #############################
 
 global:
-  # Set this to True to skip verification of server SSL certificates.  This might
-  # be useful if your Jira / Bitbucket server doesn't have a valid SSL certificate.
+  # Set this to True to skip verification of server SSL certificates.
+  # This option is deprecated; see the README.md for how to trust custom certificates.
```

Comment:
I might ask that we not go so far as to say it's "deprecated"... we don't really want to sound prescriptive on this to current or future customers :)

Reply:
Thanks for adjusting!